src/Http/IsResponse.php

```diff
@@ -63,6 +63,48 @@ public function status(Status $status): self
         return $this;
     }
 
+    public function sendHeaders(): self
```

Comment:
I don't think a response should send itself. The response is a data object, sending that response is someone else's responsibility. It doesn't have to be in `HttpApplication`, it can be a separate class, but it should not be within Response classes

Reply:
I thought this might happen. 🙃 

What do you think about a [`ResponseEmitter`](https://github.com/slimphp/Slim/blob/4.x/Slim/ResponseEmitter.php) similar to `Slim`?

Reply:
Sounds perfect 👍